js/bip32.js

```diff
@@ -57,6 +63,8 @@ BIP32.prototype.init_from_bytes = function(bytes) {
     var is_public = 
```

Comment:
why not `= !is_private`?

Reply:
I don't want to refactor and add new features in the same pull request. This can be done in another pr.

Reply:
yeah, sure, that makes a lot of sense. I should create another issue for that...  :-)